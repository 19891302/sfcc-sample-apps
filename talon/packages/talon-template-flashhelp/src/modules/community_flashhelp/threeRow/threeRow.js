import { LightningElement } from 'lwc';

export default class ThreeRow extends LightningElement {}