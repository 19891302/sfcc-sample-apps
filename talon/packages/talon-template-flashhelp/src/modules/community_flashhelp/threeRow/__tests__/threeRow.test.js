import { createElement } from 'lwc';
import threeRow from 'community_flashhelp/threeRow';

describe('community_flashhelp/threeRow', () => {
    beforeEach(() => {
        while (document.body.firstChild) {
            document.body.removeChild(document.body.firstChild);
        }
    });

    it('renders correctly', () => {
        const element = createElement('community_flashhelp-three-row', { is: threeRow });
        document.body.appendChild(element);

        return Promise.resolve().then(() => {
            expect(element).toMatchSnapshot();
        });
    });
});