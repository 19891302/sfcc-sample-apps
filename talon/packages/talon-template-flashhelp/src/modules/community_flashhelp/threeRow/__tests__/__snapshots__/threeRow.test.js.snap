// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`community_flashhelp/threeRow renders correctly 1`] = `
<community_flashhelp-three-row>
  #shadow-root(open)
    <slot
      name="header"
    />
    <slot />
    <slot
      name="footer"
    />
</community_flashhelp-three-row>
`;
